Attempt to fix Nature's Swiftness and Presence of Mind

HojNotFound opened the pull request:
I would like to start from the point that I need some help, so affected areas are:
1. Nature's Swiftness now appears for Balance spec <img width="892" height="144" alt="image" src="https://github.com/user-attachments/assets/ccbb9135-8f31-4513-a066-1532a8b9513e" />
2. But Presence of Mind isn't in talent section, and other talents are also not present, except of two <img width="927" height="444" alt="image" src="https://github.com/user-attachments/assets/761bd017-c89f-446d-be8b-0568543d0ca4" />
3. Nature's Swiftness is NOT on track when druid presses it for some reason

I need advice how can I troubleshoot these issues? Every help is very appreciated!

Commit:
Fixed location for Nature's Swiftness and Presence of Mind

vendethiel commented:
> Nature's Swiftness is NOT on track when druid presses it for some reason

Usually that’s because the spell id is a npc spell id, they just left the old spell there but it’s unused. Usually on i.e. wowhead you’ll see « uncategorized »… but sometimes not.

vendethiel commented:
…this is also why I have all classes lvled on retail and usually classic, to test spell ids and interactions…

vendethiel commented:
From a glance at wowhead, the spell id looks correct. If you have a druid, can you try to check what the spell id is?

HojNotFound commented:
From replay I have, there is a prove that it was pressed, but cooldown wasn't triggered.
<img width="807" height="124" alt="image" src="https://github.com/user-attachments/assets/0acee223-c57b-40e0-b1bd-ca1b3b03e6d4" />

I tried to do `/run print(select(7, GetSpellInfo("Nature's Swiftness")))` on my druid, it says the same id that we have in the file.

updated*
One more :c
<img width="842" height="186" alt="image" src="https://github.com/user-attachments/assets/8f4fd478-6e48-4069-ba7e-5e21dd0be281" />

updated*
I also found replay vs restoration shaman and similar ability called Ancestral Swiftness doesn't trigger cooldown.
<img width="566" height="136" alt="image" src="https://github.com/user-attachments/assets/74202bac-ef18-426f-8a03-b4ed2c3365b0" />

I believe the problem might be in the `cooldown_starts_on_aura_fade` flag

HojNotFound committed:
Fix Ironbark cooldown

vendethiel commented:
> Nature's Swiftness is NOT on track when druid presses it for some reason

From my tests, I get the spell marked as used but only when it goes off. it never gets marked as "used" like it should

vendethiel commented:
Maybe 4set has a diff id too

HojNotFound commented:
> > Nature's Swiftness is NOT on track when druid presses it for some reason
> 
> From my tests, I get the spell marked as used but only when it goes off. it never gets marked as "used" like it should

Oh, I should admit that's interesting, because for me it never ever happened. Is it possible to check arena frames functionality without arena game?
Can I also attach link with a video with my behavior?

vendethiel commented:
`/gex test3` but that will not provide a spell id (with default tooltips)

HojNotFound commented:
Yeah, that's what I do too :)
I meant player cooldowns are easy to track, cuz you do /gex test3, pressing your buttons and see what's going on.
Party cooldowns are a bit harder, cuz you need a victim you'll ask to press something :)
But arena cooldowns it's a pain, cuz you need to play arena game or ask 3 people to queue war game.
